server/chat-plugins/smogtours.ts

```diff
@@ -0,0 +1,428 @@
+/**
+ * Integration for Smogon tournaments.
+ * @author mia-pi-git
+ */
+import {FS, Utils} from '../../lib';
+
+interface TourEvent {
+	title: string;
+	url: string;
+	desc: string;
+	image?: {url: string, dimensions: [number, number]};
+	id: string;
+	shortdesc: string;
```

Comment:
```suggestion
	shortDesc: string;
```
im a camelcase loser